app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -99,6 +99,12 @@ export default {
               transaction.tx.value.msg[0].value.validator_addr
             ]
         }))
+  },
+  async mounted() {
+    this.$store.dispatch(`getTotalRewards`)
+    this.yourValidators.forEach(validator => {
+      this.$store.dispatch(`getRewardsFromValidator`, validator.operator_address)
```

Comment:
not sure about making multiple requests for this. should we request a new endpoint for this?

Reply:
imo we should eventually use something like graphQL